clients/src/main/java/org/apache/kafka/clients/consumer/KafkaConsumer.java

```diff
@@ -938,6 +947,7 @@ public void commitSync(final Map<TopicPartition, OffsetAndMetadata> offsets) {
         acquire();
         try {
             coordinator.commitOffsetsSync(offsets);
+            interceptors.onCommit(offsets);
```

Comment:
Ah, I see it now. However, it seems a little unfortunate to have to split this concern across two classes. Maybe an alternative would be to wrap the user's OffsetCommitCallback so that we could keep the interceptor in this class?